### PR TITLE
Initial update for v4 in quickstart guide

### PR DESCRIPTION
**Pull request by mrclauss**
Initial quickstart guide update for German and English

### CODE REVIEW

**Review by vicenterusso — APPROVED** [review]

File: docs/docs/de/tutorials/quickstart.md [hunk 1]
```diff
@@ -36,21 +35,7 @@ class User {
...
 IDs identifizieren Objekte in einer Collection eindeutig und erlauben es dir, sie später wiederzufinden.
 
-## 3. Code-Generator ausführen
-
-Führe den folgenden Befehl aus, um den `build_runner` zu starten:
-
-```
-dart run build_runner build
-```
-
```
Comment (Bohne13) [inline docs/docs/de/tutorials/quickstart.md:46]: I think, these lines where deleted accidentally!
It's still necessary to run 
```shell
dart run build_runner build
```
only the Flutter part can be deleted so
```shell
flutter pub run build_runner build
```
isn't needed anymore.

**Review by Bohne13 — COMMENTED** [review]
This changes should be made in all language files!

**Comment by mrclauss** [comment]
> This changes should be made in all language files!

Sure, I just wanted to get feedback (like yours below) before modifying all files. Feel free to create a pull request that adapts all languages and your objection comment. I'm trying to fix the release build right now, so I can't do that at the moment...

File: docs/docs/de/tutorials/quickstart.md (hunk 1 above)
Comment (mrclauss) [reply docs/docs/de/tutorials/quickstart.md:46]: You might be right. I ran that all the time to regenerate the files, but didn't think it's absolutely necessary.

**Comment by mrclauss** [comment]
I'll fix this now, I want to remove the manual versions from the docs...

**Pull request merged**